packages/experimental/RadioGroup/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@fluentui-react-native/experimental-radio-group",
+  "version": "0.1.0",
+  "description": "A cross-platform RadioGroup component using the Fluent Design System. Currently only implemented in win32.",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "clean": "fluentui-scripts clean",
+    "depcheck": "fluentui-scripts depcheck",
+    "just": "fluentui-scripts",
+    "lint": "fluentui-scripts eslint",
+    "test": "fluentui-scripts jest",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "prettier": "fluentui-scripts prettier",
+    "prettier-fix": "fluentui-scripts prettier --fix true"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui-react-native.git",
+    "directory": "packages/experimental/RadioGroup"
+  },
+  "dependencies": {
+    "@uifabricshared/foundation-compose": "^1.12.1",
+    "@fluentui-react-native/adapters": ">=0.9.0 <1.0.0",
+    "@fluentui-react-native/experimental-activity-indicator": "^0.6.0",
+    "@fluentui-react-native/experimental-shadow": "0.2.10",
+    "@fluentui-react-native/framework": "0.8.0",
+    "@fluentui-react-native/icon": "0.13.3",
+    "@fluentui-react-native/interactive-hooks": ">=0.18.0 <1.0.0",
+    "@fluentui-react-native/pressable": ">=0.9.6 <1.0.0",
+    "@fluentui-react-native/styling-utils": ">=0.4.1 <1.0.0",
+    "@fluentui-react-native/theme-tokens": ">=0.19.0 <1.0.0",
+    "@fluentui-react-native/theme-types": ">=0.18.0 <1.0.0",
+    "@fluentui-react-native/text": ">=0.13.0 <1.0.0",
+    "@fluentui-react-native/tokens": ">=0.16.0 <1.0.0",
+    "@fluentui-react-native/use-styling": ">=0.9.0 <1.0.0",
+    "@uifabricshared/foundation-composable": ">=0.11.0 <1.0.0",
+    "@uifabricshared/foundation-settings": ">=0.12.0 <1.0.0",
```

Comment:
@uifabricshared packages are deprecated, we shouldn't be using them in the new components.

Reply:
Forgot to remove these, fixed!